lang/cs/Org.Apache.REEF.Driver/Bridge/ClrSystemHandlerWrapper.cs

```diff
@@ -165,12 +166,21 @@ public static void Call_ClrSystemCompletedEvaluator_OnNext(ulong handle, IComple
 
         public static void Call_ClrSystemHttpServer_OnNext(ulong handle, IHttpServerBridgeClr2Java clr2Java)
         {
-            using (LOGGER.LogFunction("ClrSystemHandlerWrapper::Call_ClrSystemHttpServer_OnNext"))
+            using (LOGGER.LogFunction("ClrSystemHandlerWrapper::Call_ClrSystemHttpServer_OnNext xxx"))
             {
-                GCHandle gc = GCHandle.FromIntPtr((IntPtr)handle);
-                ClrSystemHandler<IHttpMessage> obj = (ClrSystemHandler<IHttpMessage>)gc.Target;
-                obj.OnNext(new HttpMessage(clr2Java));
-            }      
+                try
```

Comment:
I did not change this code. Maybe we need another JIRA to capture these needed refactoring? E.g. go through all code and search for the `throw` keyword and replace its use?